### PR TITLE
Feature/admin user management

### PR DESCRIPTION
**Pull request by minza55113151**
PunchPunchPunch

### CODE REVIEW

**Comment by lazykern** [comment]
👏🏿👏🏿👏🏿👏🏿

File: src/modules/admin/setting/userManagement/components/UserList.tsx [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+        </StyledTag>
+      ),
+    },
+    {
+      key: "action",
+      title: "ดำเนินการ",
+      width: "20%",
+      render: (_: string, record: TableRow) => {
+        if (record.role === roleToRoleNumber["Owner"]) return null;
+
```
Comment (GolfGrab) [inline src/modules/admin/setting/userManagement/components/UserList.tsx:273]: only disable action in current user row

**Review by GolfGrab — CHANGES_REQUESTED** [review]
nice work 😘 
have one request change

**Review by GolfGrab — APPROVED** [review]
Good Job👍👍👍

**Pull request merged**